python/flask/app/processmon.toml

```
@@ -0,0 +1,11 @@
+[[paths_to_watch]]
+path = "/app"
+
+[processes.celery]
```

nit: i don't think this process name here matters for anything but

```suggestion
[processes.flask]
```